Profiles/cio-Condition.xml

```diff
@@ -52,24 +52,19 @@
       <path value="Condition.extension" />
       <min value="1" />
     </element>
-    <element id="Condition.extension:conditionStatusDate">
+    <element id="Condition.extension:StatusDate">
       <path value="Condition.extension" />
-      <sliceName value="conditionStatusDate" />
+      <sliceName value="StatusDate" />
       <type>
         <code value="Extension" />
         <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-ConditionStatusDate" />
       </type>
     </element>
-    <element id="Condition.extension:conditionStatusDate.value[x]">
+    <element id="Condition.extension:StatusDate.value[x]">
       <path value="Condition.extension.value[x]" />
       <mapping>
         <identity value="cio-dataset-200-beta2-20231214" />
-        <map value="cio-dataelement-20-807" />
-        <comment value="ConditionStatusDate" />
-      </mapping>
-      <mapping>
-        <identity value="cio-dataset-200-beta2-20231214" />
-        <map value="cio-dataelement-20-791" />
+        <map value="cio-dataelement-20-908" />
```

Comment:
Between the beta.1 and beta.2 release all mappings have been removed, since the data set was completely renewed. Since beta.3 will have slightly groundbreaking changes w.r.t. beta.2, I'd suggest to keep the mappings to the beta.2 data set in place, and add new mappings to the beta.3 release. There is a script (which needs to be tweaked) to automatically add beta.3 mappings. By using this script, it is then only necessary to manually change the `.mapping.map` element afterwards, or remove the mapping altogether.

In the MedicationProcess profiles we follow the approach as described on https://informatiestandaarden.nictiz.nl/wiki/mp:V3.0.0_MedicationProcess9_FHIR#Mappings_between_profiles_and_data_set. By following this approach, it becomes much clearer for implementers on what elements changes have been applied.